frontend/src/views/files/Preview.vue

```diff
@@ -346,8 +346,7 @@ export default {
     close() {
       this.$store.commit("updateRequest", {});
 
-      let uri = url.removeLastDir(this.$route.path) + "/";
-      this.$router.push({ path: uri });
+      history.back();   
```

Comment:
We should use history.back() here. Then when we click the "back" button in the browser, we will not revisit all the files we previewed. The windows explorer works in the same way.